### PR TITLE
Add npm installation to quickstart guide

### LINKED ISSUE #20160: Add proper documentation for `create-directus-project`
This is a little wrapper which allows boilerplating a Directus project via npm using `npm init directus-project`. While npm installation is not an advised path, we should still surface this as part of our new getting started learning path, perhaps as part of the first step - Installation.

### PR DESCRIPTION
**Pull request by estheragbaje**
This adds a tab to the quickstart guide, explaining how to use the npm command to start up a Directus project locally.

Closes #20160

### CODE REVIEW

File: docs/getting-started/quickstart.md [hunk 1]
```diff
@@ -46,6 +46,21 @@ directus/directus
 
 _Learn more about self-hosting Directus in our [Self-Hosted Quickstart](/self-hosted/quickstart)._
```
Comment (phazonoverload) [inline docs/getting-started/quickstart.md:48]: I think `Docker` has to feature in the previous item to contextualize

File: docs/getting-started/quickstart.md [hunk 2]
```diff
@@ -46,6 +46,21 @@ directus/directus
...
+
+To start up Directus locally, run `npx directus start`.
+
+::: danger Docker is Recommended for Production
+
+While this method is suitable for quick tests and prototypes, it is not recommended for production. We recommend you use
+[Docker](/self-hosted/quickstart) because it removes environment-specific quirks that can lead to Directus not running
+properly.
+
 :::
```
Comment (phazonoverload) [inline docs/getting-started/quickstart.md:64]: I'm not sold on this narrative. It's not a non-prod/prod split, it's a potential complexity split. 

```suggestion
You can initialize a new Directus project using the CLI command:
`npm init directus-project`.

Follow prompts to configure your database and create your first admin user.

To start Directus locally, run `npx directus start`.

:::danger Docker is Recommended

While this method works and is maintained, it is not recommended. Docker removes environment-specific quirks that can lead to Directus not running
properly in some contexts. 

:::
```

Removing link to Self-Hosted Quickstart too as it's the previous tab in the box this text displays in. We'll probably want @rijkvanzanten to chime in with desired wording too.

**Review by phazonoverload — CHANGES_REQUESTED** [review]

File: docs/getting-started/quickstart.md (hunk 2 above)
Comment (rijkvanzanten) [reply docs/getting-started/quickstart.md]: The current state LGTM!

**Comment by phazonoverload** [comment]
@estheragbaje can you add a changeset?

**Review by phazonoverload — APPROVED** [review]

**Review by paescuj — APPROVED** [review]

**Pull request merged**